src/Fist/Database/tests/DatabaseConnectionTest.php

```diff
@@ -136,7 +136,9 @@ public function testConnectionSwapping()
     public function testDriverConnectionAreShared()
     {
         $db = new Database(
-            $repository = new ContainerRepository($container = new Container(), 'db.')
+            new ContainerRepository(
+                new Container(), 'db.'
```

Comment:
__[code_sniffer]__
* Only one argument is allowed per line in a multi-line function call

<!-- Comment by [SideCI](https://sideci.com) -->